templates/publishing/src/blog-app.html

```diff
@@ -40,6 +40,7 @@
 
       :host {
         --app-primary-color: #404040;
+        -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
```

Comment:
I think app-layout is in a more experimental stage, so it's safe to remove it here. I would leave it in the paper-elements for now since it's not too burdensome right now.